src/pudl/metadata/resources/allocate_net_gen.py

```diff
@@ -0,0 +1,196 @@
+                "net_generation_mwh",
+            ],
+            "primary_key": [
+                "report_date",
+                "plant_id_eia",
+                "generator_id",
+            ],
+        },
+        "field_namespace": "eia",
+        "sources": ["eia923", "eia860"],
+        "etl_group": "outputs",
+    },
+    # "generation_fuel_by_generator_energy_source_owner_monthly_eia923": {
+    #     "description": (
+    #         "Monthly estimated net generation and fuel consumption for each generator, "
+    #         "broken down by energy source, prime mover, and owner. Based on data "
+    #         "reported in the EIA-923 generation and generation_fuel tables."
+    #     ),
+    #     "schema": {
+    #         "fields": [
+    #             "report_date",
+    #             "plant_id_eia",
+    #             "generator_id",
+    #             "prime_mover_code",
+    #             "energy_source_code",
+    #             "utility_id_eia",
+    #             "ownership_record_type",
+    #             "fraction_owned",
+    #             "capacity_mw",
+    #             "energy_source_code_num",
+    #             "net_generation_mwh",
+    #             "fuel_consumed_mmbtu",
+    #             "fuel_consumed_for_electricity_mmbtu",
+    #         ],
+    #         "primary_key": [
+    #             "report_date",
+    #             "plant_id_eia",
+    #             "generator_id",
+    #             "prime_mover_code",
+    #             "energy_source_code",
+    #             "utility_id_eia",  # This is the OWNER not the operator.
+    #             "ownership_record_type",
+    #         ],
+    #     },
+    #     "field_namespace": "eia",
+    #     "sources": ["eia923", "eia860"],
+    #     "etl_group": "outputs",
+    # },
+    "generation_fuel_by_generator_energy_source_owner_yearly_eia923": {
+        "description": (
+            "Yearly estimated net generation and fuel consumption for each generator, "
+            "broken down by energy source, prime mover, and owner. Based on data "
+            "reported in the EIA-923 generation and generation_fuel tables."
+        ),
+        "schema": {
+            "fields": [
+                "report_date",
+                "plant_id_eia",
+                "generator_id",
+                "prime_mover_code",
+                "energy_source_code",
+                "utility_id_eia",
+                "ownership_record_type",
+                "fraction_owned",
+                "capacity_mw",
+                "energy_source_code_num",
+                "net_generation_mwh",
+                "fuel_consumed_mmbtu",
+                "fuel_consumed_for_electricity_mmbtu",
+            ],
+            "primary_key": [
+                "report_date",
+                "plant_id_eia",
+                "generator_id",
+                "prime_mover_code",
+                "energy_source_code",
+                "utility_id_eia",  # This is the OWNER not the operator.
```

Comment:
this feels like a note that is relevant to add into the table description or to overwrite in the table-specific field description overwriting place

Reply:
alternatively we could keep the id name as the owner name ( i think we are currently renaming the owner id ). I believe this rename was requested by someone at rmi bc it makes the a little more seamless to use in conjunction with other tables so I am on the fence about that.

Reply:
I've added note into the table description at least. Added to the [naming revamp issue](https://github.com/catalyst-cooperative/pudl/issues/2533)